lnrpc/routerrpc/router_server.go

```diff
@@ -449,6 +454,9 @@ func (s *Server) trackPayment(paymentHash lntypes.Hash,
 		}
 	}
 
+	s.wg.Add(1)
```

Comment:
Still this feels a bit strange. Normally we add to the waitgroup right before we start the goroutine. In this case, grpc started the goroutine already, so are we too late here? I don't know the answer really, but maybe it doesn't make sense after all to do this?

Reply:
On the router server level, adding this wait group means that we don't consider the server to be stopped before all the subscriptions have received the quit signal. So I think that adds something, although all the other subservers are shutting down fine without it, so also unsure.